Makefile

```diff
@@ -26,19 +27,24 @@ test-protocols: build-s3-browser-bundle
 	yarn g:vitest run -c vitest.config.protocols.integ.ts
 
 test-integration: build-s3-browser-bundle
+	build-signature-v4-multi-region-browser-bundle
 	rm -rf ./clients/client-sso/node_modules/\@smithy # todo(yarn) incompatible redundant nesting.
 	yarn g:vitest run -c vitest.config.integ.ts
 	npx jest -c jest.config.integ.js
 	make test-protocols;
 	make test-types;
 
 test-e2e: build-s3-browser-bundle
+	build-signature-v4-multi-region-browser-bundle
```

Comment:
this is a step dependency and can look like this:

```mk
test-e2e: build-s3-browser-bundle build-signature-v4-multi-region-browser-bundle
	yarn ...
```

---- 

for less words:

```mk
.PHONY: bundles test-unit test-integration test-protocols test-e2e

bundles: build-s3-browser-bundle build-signature-v4-multi-region-browser-bundle

test-e2e: bundles
	yarn ...

```